tools/yarn_install.R

```diff
@@ -11,7 +11,8 @@ if (Sys.which("yarn") == "") {
   stop("The yarn CLI must be installed and in your PATH")
 }
 
-withr::with_dir("inst", system("yarn install"))
+# only install the direct deps
+withr::with_dir("inst", system("yarn install --production"))
```

Comment:
Are we sure that this is the right way to go?

Reply:
This makes it so we can use `devDependencies` without having to clean-up parcel